test/index.test.ts

```diff
@@ -250,6 +258,44 @@ describe('Successful tests', () => {
         ).toMatchSnapshot();
     });
 
+    it('All optional parameters present, serving resources', async () => {
+        const reportFileName = 'tcServingResources.html';
```

Comment:
@lpelypenko the [artifacts directory is ignored](https://github.com/lpelypenko/axe-html-reporter/blob/0f6b7832690708716b40a7113f1b7c8561cdefba/.gitignore#L113) - so `tcServingResources.html` will not be tracked, but other test artifacts will. Was not sure if this is desired behavior.